### ISSUE TITLE
Build Angular version 17 for @trimble-oss/modus-angular-components

### ISSUE THREAD

**Issue by quynh-le-trimble** [trimble-oss/modus-web-components#2002]
### Prerequisites

- [X] I have [searched](https://github.com/trimble-oss/modus-web-components/issues?q=is%3Aissue) for duplicate or closed feature requests
- [X] I have read the [Modus style guidelines](https://modus.trimble.com/components/web/introduction/)

### Proposal

Build Angular version 17 for @trimble-oss/modus-angular-components

### Motivation and context

For Angular project v17+

### What is the issue regarding ?

@trimble-oss/modus-angular-components

### Priority

High

### What product/project are you using Modus Components for ?

Prism 

### What is your team/division name ?

Prism

### Are you willing to contribute ?

Yes

### Are you using Modus Web Components in production ?

No

**Comment by coliff** [comment]
Hey there @quynh-le-trimble - it'd be great if you wanted to take this on! Let me know if you have any questions about it.

**Comment by quynh-le-trimble** [comment]
@coliff Sure, is there any documentation available where i can get these changes updated and published?

**Comment by coliff** [comment]
Hello! I added the 'angular-16' and these were the equivalent steps I took to do that.

- Made of a copy of the `modus-web-components\angular-workspace\ng15` folder and name it ng16 (in your case it'd be copying the ng16 folder and then name of the new folder ng17).
- In the `package.json` in that folder, update engines (node) to `>=18.13.0` 
- Read through the upgrade guide: https://update.angular.io/?v=16.0-17.0
